var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -164,6 +175,45 @@ def setup_build_environment(self, env):
         elif self.spec.satisfies("%fj +fortran"):
             env.set("FFLAGS", "-Ccpp")
 
+    @property
```

Comment:
Why moving this two functions blocks? If considered important for the consistency of the file, please do it in a independent commit

Reply:
This is an artifact of the significant amount of history manipulation this file has been exposed to, it can/will be removed.